lib/qmp.ml

```diff
@@ -162,6 +182,24 @@ let message_of_string str =
       let filename = json |> arguments |> U.member "filename" |> U.to_string in
       (device, filename)
     in
+    let device_del json =
+      json |> arguments |> U.member "id" |> U.to_string
+    in
+    let device_add json =
+      let driver = json |> arguments |> U.member "driver" |> U.to_string in
+      let id     = json |> arguments |> U.member "id"     |> U.to_string in
+      let maybe_mem_of k x = x |> U.member k |> U.to_option U.to_string in
+      let params = json |> arguments |> fun x ->
+      match maybe_mem_of "bus" x, maybe_mem_of "hostbus" x, maybe_mem_of "hostport" x with
```

Comment:
Please increase the level of indentation of this `match` block

Reply:
updated.